src/mockLink.ts

```diff
@@ -23,38 +24,45 @@ export class MockLink extends ApolloLink {
     this.requestHandlers[key] = handler;
   }
 
-  request = (operation: Operation) =>
-    new Observable<FetchResult>(observer => {
-      const key = requestToKey(operation.query);
+  request = (operation: Operation) => {
+    const key = requestToKey(operation.query);
 
-      const handler = this.requestHandlers[key];
+    const handler = this.requestHandlers[key];
 
-      if (!handler) {
-        throw new Error(`Request handler not defined for query: ${print(operation.query)}`);
-      }
+    if (!handler) {
+      throw new Error(`Request handler not defined for query: ${print(operation.query)}`);
```

Comment:
Hi @Sytten @Mike-Gibson 
Throwing an error directly from a function body and not from the returned observable is a not backward-compatible change of the API and it was introduced in the minor version of the package(1.1.0). 
Could you please tell me if it was done intentionally? If not, I could contribute the fix for it.
Thanks

Reply:
No idea its been too long, I dont work with apollo anymore